### ISSUE TITLE
Smart mirror does not respond to microphone

### ISSUE THREAD

**Issue by kermit-kr** [evancohen/smart-mirror#448]
There are no errors in all sources
USB microphone not recognized

This is the situation now.

```
pi@raspberrypi:~/smart-mirror $ aplay -l
**** List of PLAYBACK Hardware Devices ****
card 0: ALSA [bcm2835 ALSA], device 0: bcm2835 ALSA [bcm2835 ALSA]
  Subdevices: 7/8
  Subdevice #0: subdevice #0
  Subdevice #1: subdevice #1
  Subdevice #2: subdevice #2
  Subdevice #3: subdevice #3
  Subdevice #4: subdevice #4
  Subdevice #5: subdevice #5
  Subdevice #6: subdevice #6
  Subdevice #7: subdevice #7
card 0: ALSA [bcm2835 ALSA], device 1: bcm2835 ALSA [bcm2835 IEC958/HDMI]
  Subdevices: 1/1
  Subdevice #0: subdevice #0

pi@raspberrypi:~/smart-mirror $ arecord -l
**** List of CAPTURE Hardware Devices ****
card 1: Device [USB PnP Sound Device], device 0: USB Audio [USB Audio]
  Subdevices: 1/1
  Subdevice #0: subdevice #0

pi@raspberrypi:~/smart-mirror $ cat ~/.asoundrc 
pcm.!default {
        type asym
        playback.pcm{
                type hw
                card 0
        }
        capture.pcm{
                type hw
                card 1
        }
}

ctl.!default {
        type asym
        playback.pcm{
                type hw
                card 0
        }
}

```
It works in annyang.simulate but does not recognize the microphone.
Sounds like a problem with microphone input.

### Your Environment

* Environment name and version (e.g. Raspberry Pi 3, node.js 5.4): Raspberry pi 3, node.js 6.9.1
* Operating System and version (e.g. Raspbian, Ubuntu, OSX): Raspbian

**Comment by duckhee** [comment]
me too doesn' work microphone......  I sugest snowboy doesn't work maybe.....

**Comment by justbill2020** [comment]
It's an issue with your configuration... @kermit-kr

Here's the `~/.asoundrc` based on the info above... this is if your plugging into the 3.5mm jack... FYI search for a PlayStation Eye to use for a microphone... I've been noticing issues with USB microphones that use that driver... the audio is extremely low volume wise.

```
pcm.!default {
  type asym
   playback.pcm {
     type plug
     # This is your output device (In this case AUX out on the Pi)
     slave.pcm "hw:0,0"
   }
   capture.pcm {
     type plug
     # This is your input device (it may be different from what is seen here)
     slave.pcm "hw:1,0"
   }
}
```

**Comment by kermit-kr** [comment]
@justbill2020
I tried that but it does not work.

I do not know how ...

**Comment by justbill2020** [comment]
This is different than what you posted previously...

**Comment by justbill2020** [comment]
99% of the time it is during this step or other troubleshooting breaking something else... I should have time tomorrow to help troubleshoot this.

**Comment by kermit-kr** [comment]
I am trying to output sound using hdmi and I am trying to input sound using a usb microphone.

But the microphone is not recognized.

Even using the above method, it is not recognized at all.

Help.

**Comment by evancohen** [comment]
Can you try `rec test.wav` and record a sound sample? If you get an error like `rec - command not recognized` then you are missing dependencies.  If you don't see the bars moving it's a config issue. If you see the bars moving then it should work just fine.

**Comment by kermit-kr** [comment]
Hi~ @evancohen 

Thanks for the answer.

but I have suffered a great trial.

The bar moves when you try rec test.wav.

I tried recording through the microphone. It does not work on smart mirrors.

Everything works when annyang.trigger is done, but the mic does not work.

I have tried using the logitch c110 cam because it is a microphone problem but it does not work the same.

What should I do now? Help me.
